lib/zosung.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function nextSeq(entity) {
```

Comment:
Please move all code to `devices/aubess.js`

Reply:
I left it in the file, but now it is a local function.